pkg/os/reboot_test.go

```diff
@@ -2,11 +2,16 @@ package reboot
 
 import (
 	"context"
+	"os"
 	"testing"
 	"time"
 )
 
 func TestReboot(t *testing.T) {
+	if os.Geteuid() == 0 { // running as root
```

Comment:
This test is invalid.. we can mock a reboot command like `echo rebooted` to test the function.